app/views/jobs/index.html.erb

```diff
@@ -29,10 +31,12 @@
                           <ul>
                             <li><%= link_to job.job_author, job.url %></li>
                           </ul>
-                        <div class="pv2 f6">
-                          <%= sanitize(job.description.truncate(200, separator: '</p>')) %>
+                          
+                          <div class="pv2 f6">
+                          <%= truncate(strip_tags(job.description.body.to_s), length: 30) %>
                         </div>
```

Comment:
these lines are not indented correctly. Can you fix this first?
Also check all the pages if they are indented correctly.